setup.py

```diff
@@ -138,12 +139,12 @@ def _set_manifest_path(manifest_dir: str, aggregate: bool = False) -> Generator:
             raise ValueError(
                 f"Unexpected package name: {_PACKAGE_NAME}. Possible choices are: {list(_PACKAGE_MAPPING)}"
             )
-        package_to_install = _PACKAGE_MAPPING.get(_PACKAGE_NAME, "lightning")
-        if package_to_install == "lightning":  # install everything
+        if _PACKAGE_NAME in _PACKAGE_STANDALONE:  # install everything
             # merge all requirements files
             assistant._load_aggregate_requirements(_PATH_REQUIRE, _FREEZE_REQUIREMENTS)
```

Comment:
This step should only be done if it's installing "lightning", but these changes now do it also when `PACKAGE_NAME=app`

Reply:
This is what I was trying to address with [`d4dba2f` (#16553)](https://github.com/Lightning-AI/lightning/pull/16553/commits/d4dba2fa40a489ad156b154962fa196b9b06f431)

Reply:
Which is correct, as it is what the reverse steps is about

Reply:
but yes, the manifest can be split if, but no special need, changed in [20d9652](https://github.com/Lightning-AI/lightning/pull/16553/commits/20d9652cd9aeee2adf52b1d0fdbf10b3db2a7d1b)